docs/reference/configuration/keywords.rst

```diff
@@ -677,43 +677,145 @@ FIELD
 -----
 .. _field:
 
-The FIELD keyword is used to parametrize quantities which have extent over the
-full grid. In order to use the FIELD keyword, the GRID keyword must be supplied.
+The ``FIELD`` keyword is used to parametrize quantities that span the entire grid,
+with porosity and permeability being the most common examples.
+In order to use the ``FIELD`` keyword, the :ref:`GRID<grid>` keyword must be supplied.
 
-A parameter field (e.g. porosity or permeability or Gaussian Random Fields from APS) is defined as follows:
+Field parameters (e.g. porosity, permeability or Gaussian Random Fields from APS) are defined as follows:
 
 ::
 
-        FIELD  ID PARAMETER   <ECLIPSE_FILE>  INIT_FILES:/path/%d  MIN:X MAX:Y OUTPUT_TRANSFORM:FUNC INIT_TRANSFORM:FUNC  FORWARD_INIT:True
+        FIELD  ID  PARAMETER  <ECLIPSE_FILE>  INIT_FILES:/path/%d  FORWARD_INIT:True  INIT_TRANSFORM:FUNC  OUTPUT_TRANSFORM:FUNC  MIN:X  MAX:Y
 
-Here ID must be the same as the name of the parameter in the INIT_FILES.
-ECLIPSE_FILE is the name of the file ERT will export this field to when
-running simulations. Note that there should be an IMPORT statement in
-the ECLIPSE data file corresponding to the name given with ECLIPSE_FILE in case
-the field parameter is a field used in ECLIPSE data file like perm or poro.
-INIT_FILES is a filename (with an embedded %d if FORWARD_INIT is set to False)
-to load the initial field from. Can be RMS ROFF format, ECLIPSE restart format
-or ECLIPSE GRDECL format.
+- **ID**
+  Arbitrary string identifier that must match the name of the parameter specified in ``INIT_FILES``.
 
-FORWARD_INIT:True means that the files specified in the INIT_FILES are expected
-to be created by a forward model, and does not need any embedded %d.
-FORWARD_INIT:False means that the files must have been created before running
-ERT and need an embedded %d.
+- **PARAMETER**
+  Legacy from when ERT supported EnKF and needed to handle dynamic fields like pressure and saturations.
 
-The input arguments MIN, MAX, INIT_TRANSFORM and OUTPUT_TRANSFORM are all
-optional.
+- **ECLIPSE_FILE**
```

Comment:
Suggest renaming to for example: `<OUTPUT_FILE>`, as this is the output file that ert creates on `runpath`, can go into the simulator, or it may be read by rms.